pkg/providers/imagefamily/bootstrap/aksbootstrap.go

```diff
@@ -143,7 +143,7 @@ type NodeBootstrapVariables struct {
 	ContainerRuntime                  string   // s   always containerd
 	CLITool                           string   // s   static/unnecessary
 	ContainerdDownloadURLBase         string   // -   unnecessary
-	NetworkMode                       string   // c   user input
+	NetworkMode                       string   // o   user input
```

Comment:
Why change these all to o? Perosonally i thought c for cluster level is was clear enough. 

I agree with changing comment to imply they come from options

Reply:
I wasn't sure if `c` stands for `Cluster` for `ConfigMap` either. I thought it was `ConfigMap` so I changed it.
But looks like it is more for `Cluster`, so let's change it back.